tagmanager/src/tm_source_file.c

```diff
@@ -35,6 +35,14 @@
 #include "tm_source_file.h"
 #include "tm_tag.h"
 
+typedef struct
+{
+	TMSourceFile public;
+	gint refcount;
+} TMSourceFilePriv;
+
+#define SOURCE_FILE_NEW(S) ((S) = g_slice_new(TMSourceFilePriv))
+#define SOURCE_FILE_FREE(S) g_slice_free(TMSourceFilePriv, (TMSourceFilePriv *) S)
```

Comment:
these two macros might not be sensible, tmtag is like that but I'm not quite sure it helps at anything as we have only 1 call site for each, so it only adds indirection